Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/resources/Templates/ContentForBaseTemplate.xhtml

```diff
@@ -27,6 +27,7 @@
    * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
    */ 
 -->
+<!DOCTYPE html>
```

Comment:
Only for this file, use this doctype similar like this https://github.com/CONNECT-Solution/CONNECT/blob/CONNECT_integration/Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/resources/Templates/BaseTemplate.xhtml#L30